.gitignore

```diff
@@ -75,6 +75,10 @@ Thumbs.db
 doc/api/generated
 doc/build
 doc/manual
+doc/notebooks/.ipynb_checkpoints
```

Comment:
How about adding this at top level to ignore this directory at any level - as in:

```
.ipynb_checkpoints/
```